web/packages/teleport/src/Integrations/status/AwsOidc/Details/Details.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useParams } from 'react-router';
+
+import { FeatureBox } from 'teleport/components/Layout';
+import { AwsOidcHeader } from 'teleport/Integrations/status/AwsOidc/AwsOidcHeader';
+import { AwsOidcTitle } from 'teleport/Integrations/status/AwsOidc/AwsOidcTitle';
+import { Rds } from 'teleport/Integrations/status/AwsOidc/Details/Rds';
+import { Rules } from 'teleport/Integrations/status/AwsOidc/Details/Rules';
+import { AwsResource } from 'teleport/Integrations/status/AwsOidc/StatCard';
+import { useAwsOidcStatus } from 'teleport/Integrations/status/AwsOidc/useAwsOidcStatus';
+import { IntegrationKind } from 'teleport/services/integrations';
+
+export function Details() {
+  const { resourceKind } = useParams<{
+    type: IntegrationKind;
+    name: string;
+    resourceKind: AwsResource;
+  }>();
+
+  const { integrationAttempt } = useAwsOidcStatus();
+  const { data: integration } = integrationAttempt;
+  return (
+    <>
+      {integration && (
+        <AwsOidcHeader integration={integration} resource={resourceKind} />
+      )}
+      <FeatureBox css={{ maxWidth: '1400px', paddingTop: '16px', gap: '30px' }}>
+        {integration && (
+          <AwsOidcTitle integration={integration} resource={resourceKind} />
+        )}
+        {resourceKind == AwsResource.ec2 && <Rules />}
+        {resourceKind == AwsResource.eks && <Rules />}
+        {resourceKind == AwsResource.rds && <Rds />}
```

Comment:
```suggestion
        {resourceKind == AwsResource.rds ? <Rds /> : <Rules />}
```
unless there are some other resources we are ignoring as well